finch/processes/wps_geoseries_to_netcdf.py

```diff
@@ -0,0 +1,200 @@
+from pywps import ComplexInput, ComplexOutput, FORMATS, LiteralInput
+
+import logging
+from pathlib import Path
+from .wps_base import FinchProcess
+from . import wpsio
+from .utils import log_file_path, write_log, dataset_to_netcdf
+from xclim.core.formatting import update_history
+import xarray as xr
+import numpy as np
+import cf_xarray.geometry as cfgeo
+import geopandas as gpd
+from urllib.parse import urlparse
+
+LOGGER = logging.getLogger("PYWPS")
+
+
+class GeoseriesToNetcdfProcess(FinchProcess):
+    def __init__(self):
+        inputs = [
+            ComplexInput(
+                "resource",
+                "Geospatial series",
+                abstract="Series of geographical features.",
+                supported_formats=[FORMATS.GEOJSON],
+                min_occurs=1,
+                max_occurs=1,
+            ),
+            LiteralInput(
+                "index_dim",
+                "Index dimension",
+                abstract="Name of the column in the data to be converted into the index dimension.",
+                data_type="string",
+                min_occurs=1,
+                max_occurs=1,
+            ),
+            LiteralInput(
+                "feat_dim",
+                "Feature dimension",
+                abstract="Name of the column in the data to be converted into the 'feature' dimension. Each geometry must be different along and only along this dimension.",
+                data_type="string",
+                default="",
+                min_occurs=0,
+                max_occurs=1
+            ),
+            LiteralInput(
+                "squeeze",
+                "Squeeze variables",
+                abstract="Squeeze variables that are replicated along one of the dimensions.",
+                data_type="boolean",
+                default=False,
+                min_occurs=0,
+                max_occurs=1,
+            ),
+            LiteralInput(
```

Comment:
I suggest commenting this, and add it later when more than one option is available, setting the default to the current value. This way, no code will break.